### PR TITLE
fix(scheduler): Fix deleting models that are still progressing

### PR DESCRIPTION
**Pull request by sakoush**
**What this PR does / why we need it**:

This changes allows deleting models that are still progressing when we delete the associated server. Previously the model state gets stuck.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: k8s/yaml/dcgm-on-gke/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (cliveseldon) [inline k8s/yaml/dcgm-on-gke/LICENSE:1]: Why is this file been added?

File: k8s/yaml/dcgm-on-gke/grafana/grafana.yml [hunk 2]
```diff
@@ -0,0 +1,147 @@
+# Copyright 2022 Google LLC
+#
```
Comment (cliveseldon) [inline k8s/yaml/dcgm-on-gke/grafana/grafana.yml:2]: and this folder

File: k8s/yaml/dcgm.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
```
Comment (cliveseldon) [inline k8s/yaml/dcgm.yaml:3]: and this

File: k8s/yaml/dcgm-on-gke/LICENSE (hunk 1 above)
Comment (sakoush) [reply k8s/yaml/dcgm-on-gke/LICENSE]: this was a commit from a stale branch, remove now in

File: k8s/yaml/dcgm-on-gke/grafana/grafana.yml (hunk 2 above)
Comment (sakoush) [reply k8s/yaml/dcgm-on-gke/grafana/grafana.yml]: removed

File: k8s/yaml/dcgm.yaml (hunk 3 above)
Comment (sakoush) [reply k8s/yaml/dcgm.yaml]: removed

**Review by cliveseldon — APPROVED** [review]

**Pull request merged**